### DISCUSSION TITLE
.render_svg

### DISCUSSION THREAD

**Discussion by lookme2** [Argmaster/pygerber#225] [General]
Is that available with a "Project"?

**Comment by Argmaster** [comment]
As for now, unfortunately it is not, although I may look into it in the future if such feature is desired by the community.

**Reply by lookme2** [reply]
Thanks for the info.  I am writing an app that lets you use the pick and place file to "point out" the component to hand solder.  Thanks for all the good work on this project.

**Reply by Argmaster** [reply]
I'm open for deeper cooperation in case you need some specific features to be added to PyGerber, just let me know. You can always post an Issue with feature request. If SVG rendering of projects is needed please open an Issue for that 🙂
